shared/src/ContentReactor.Shared/BlobRepository/BlobRepository.cs

```diff
@@ -1,132 +1,139 @@
-﻿using System;
+﻿using Azure;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage.Sas;
+using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.WindowsAzure.Storage;
-using Microsoft.WindowsAzure.Storage.Blob;
 
 namespace ContentReactor.Shared.BlobRepository
 {
     public interface IBlobRepository
```

Comment:
The structure of this file violates common the common C# convention of "one type per file"."    We may want to consider splitting these interfaces out into their own dedicated files to better snap to norms.

Reply:
Resolved